great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -798,75 +791,6 @@ def _validate_suite_unique_constaints_before_save(self, key: GXCloudIdentifier)
                 "expectation_suite, set overwrite_existing=True."
             )
 
-    @override
-    def add_checkpoint(  # noqa: PLR0913
```

Comment:
Since all checkpoint CRUD is now done through the "factory" and these methods utilize the legacy component, we should remove them for all contexts